qlib/kernel/task.rs

```diff
@@ -165,10 +160,51 @@ impl Drop for Task {
     }
 }
 
+
+// task wrapper on shared memory
+#[repr(C)]
+pub struct TaskWrapper {
```

Comment:
Perhaps add some documentations on this?  :D

Reply:
Working on it now